hotspot/src/cpu/ppc/vm/sharedRuntime_ppc.cpp

```diff
@@ -42,7 +42,9 @@
 #include "opto/runtime.hpp"
 #endif
 
+#ifndef TARGET_OS_ARCH_bsd_ppc
 #include <alloca.h>
+#endif
```

Comment:
Sorry, I was sure I test compiled this on PPC before release :(

In sharedRuntime_x86.cpp we essentially write this as:

```
#ifdef _ALLBSD_SOURCE
#include <stdlib.h>
#else
#include <alloca.h>
#endif
```

It's a bit more complicated because there are more OSes involved, but it would be good to be consistent here.